libcontainer/container_linux.go

```diff
@@ -1890,7 +1890,10 @@ func (c *linuxContainer) currentState() (*State, error) {
 	)
 	if c.initProcess != nil {
 		pid = c.initProcess.pid()
-		startTime, _ = c.initProcess.startTime()
+		var err error
+		if startTime, err = c.initProcess.startTime(); err != nil {
```

Comment:
I don't think failing to get `startTime` is a critical error; if we return here because of that error, critical paths of the code base would no longer be able to function, so I don't think we should return an error here (at _most_ it could be logged, but perhaps it's not even important to do so)